vercel.json

```diff
@@ -0,0 +1,5 @@
+{
+  "rewrites": [
+    { "source": "/", "destination": "/docs/" }
+  ]
+}
```

Comment:
Sure that we need this?

Reply:
I think it should work without the rewrite there since all traffic will go through Cloudfront with the correct uri